tests/unittests/test_commands.py

```diff
@@ -105,5 +105,20 @@ def test_text_saves_params(self):
         self.assertEqual(output_text.fontsize, 64)
 
 
+class TestFontUtils(ShoebotTestCase):
+    @test_as_bot()
+    def test_fontnames_gives_output(self):
+        """
+        Verify that fontnames() gives a list as output.
+        """
+        output = fontnames()
+        # it's a list
+        self.assertTrue(isinstance(output, list))
+        # it's not empty
+        self.assertTrue(bool(output))
```

Comment:
assertions shouldn't change the data being asserted if possible.
If changes are needed then that should be further up,

Ideally, the test is built with code to:

```
# code to arrange data

# code to act:
# (call API being tested)
# if any changes are needed do that here

# Assert ...  do assertions here.
```

Reply:
i used bool() as a hack to determine if the list is empty or not, since I don't know better :)

i can't understand the issue with isinstance? (sorry)

Reply:
ok, fixed it by using assertTrue(list)